scripts/check_dependencies.sh

```diff
@@ -43,27 +43,22 @@ fi
 
 if which python >&/dev/null ; then
```

Comment:
I think we don't need for the default python to be python3 any more, as we are invoking it expliticly (but check via `git grep -w python | grep -v python3`).  So we could just check that python3 is installed here.

Reply:
Fixed.